pegasus/forms/hoc_signup_2017.rb

```diff
@@ -1,6 +1,34 @@
 require pegasus_dir 'forms/hoc_signup_2014'
 
 class HocSignup2017 < HocSignup2014
+
```

Comment:
I thought I could do something like the following to override the 2014 self.normalize and include the new fields for 2017, but I'm getting an error. `NameError - undefined local variable or method `result' for HocSignup2017:Class:` Is there something I'm missing?